test/internalTest.js

```diff
@@ -60,6 +60,24 @@ mineflayer.supportedVersions.forEach((supportedVersion, i) => {
         })
       })
     })
+    it('spawn event', (done) => {
```

Comment:
I believe it should be an external test. In #783 someone claimed it happens only on non-vanilla servers. We may add support for non-vanilla tests in the future.